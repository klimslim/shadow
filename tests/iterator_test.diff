     Iterating: %s/templatedir/api
002- Path: $template/api, isDir: 1, isFile: 0
     Path: $template/api/rest.php, isDir: 0, isFile: 1
004- Count for %s/templatedir/api: 2
003+ Count for /app/tests/fixtures/templatedir/api: 1

     Iterating: /app/tests/fixtures/instance/qwe
007- Path: $instance/qwe, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules, isDir: 1, isFile: 0
011+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts, isDir: 1, isFile: 0
012+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts/.gitkeep, isDir: 0, isFile: 1
013+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks, isDir: 1, isFile: 0
014+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks/.gitkeep, isDir: 0, isFile: 1
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts/.gitkeep, isDir: 0, isFile: 1
015- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls, isDir: 1, isFile: 0
016- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls/.gitkeep, isDir: 0, isFile: 1
017+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers, isDir: 1, isFile: 0
018+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers/.gitkeep, isDir: 0, isFile: 1
019+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents, isDir: 1, isFile: 0
020+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents/.gitkeep, isDir: 0, isFile: 1
021+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project, isDir: 1, isFile: 0
022+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project/.gitkeep, isDir: 0, isFile: 1
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases/.gitkeep, isDir: 0, isFile: 1
019- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts, isDir: 1, isFile: 0
020- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts/.gitkeep, isDir: 0, isFile: 1
021- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents, isDir: 1, isFile: 0
022- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents/.gitkeep, isDir: 0, isFile: 1
023- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers, isDir: 1, isFile: 0
024- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers/.gitkeep, isDir: 0, isFile: 1
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings/.gitkeep, isDir: 0, isFile: 1
027+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls, isDir: 1, isFile: 0
028+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls/.gitkeep, isDir: 0, isFile: 1
029+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems, isDir: 1, isFile: 0
030+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems/.gitkeep, isDir: 0, isFile: 1
031+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes, isDir: 1, isFile: 0
032+ Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes/.gitkeep, isDir: 0, isFile: 1
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages, isDir: 1, isFile: 0
     Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages/.gitkeep, isDir: 0, isFile: 1
029- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes, isDir: 1, isFile: 0
030- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes/.gitkeep, isDir: 0, isFile: 1
031- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project, isDir: 1, isFile: 0
032- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project/.gitkeep, isDir: 0, isFile: 1
033- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems, isDir: 1, isFile: 0
034- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems/.gitkeep, isDir: 0, isFile: 1
035- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks, isDir: 1, isFile: 0
036- Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks/.gitkeep, isDir: 0, isFile: 1
037- Count for %s/instance/qwe: 30
035+ Count for /app/tests/fixtures/instance/qwe: 29

     Iterating: /app/tests/fixtures/instance/api
040- Path: $instance/api, isDir: 1, isFile: 0
     Path: $instance/api/rest.php, isDir: 0, isFile: 1
042- Count for %s/instance/api: 2
039+ Count for /app/tests/fixtures/instance/api: 1
