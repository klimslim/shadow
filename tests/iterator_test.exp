Iterating: %s/templatedir/api
Path: $template/api, isDir: 1, isFile: 0
Path: $template/api/rest.php, isDir: 0, isFile: 1
Count for %s/templatedir/api: 2

Iterating: %s/instance/qwe
Path: $instance/qwe, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks/.gitkeep, isDir: 0, isFile: 1
Count for %s/instance/qwe: 30

Iterating: %s/instance/api
Path: $instance/api, isDir: 1, isFile: 0
Path: $instance/api/rest.php, isDir: 0, isFile: 1
Count for %s/instance/api: 2