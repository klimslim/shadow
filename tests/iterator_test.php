<?php
require_once('setup.inc'); // This likely sets $template and $instance

// Corrected paths based on created fixtures
$topLevelDirs = [
    "$template/api", // Should be tests/fixtures/templatedir/api
    "$instance/qwe", // Should be tests/fixtures/instance/qwe
    "$instance/api", // Should be tests/fixtures/instance/api
];

foreach ($topLevelDirs as $dirPath) {
    echo "Iterating: $dirPath\n";
    if (!is_dir($dirPath)) {
        echo "Directory $dirPath does not exist!\n";
        continue;
    }
    $count = 0;
    $iterator = new RecursiveIteratorIterator(
        new RecursiveDirectoryIterator($dirPath, FilesystemIterator::SKIP_DOTS | FilesystemIterator::UNIX_PATHS),
        RecursiveIteratorIterator::SELF_FIRST
    );

    foreach ($iterator as $item) {
        $count++;
        // Use a relative path for EXPECTF robustness if $template/$instance changes base
        $displayPath = $item->getPathname();
        if (strpos($displayPath, $template) === 0) {
            $displayPath = '$template' . substr($displayPath, strlen($template));
        } elseif (strpos($displayPath, $instance) === 0) {
            $displayPath = '$instance' . substr($displayPath, strlen($instance));
        }
        // Normalize slashes
        $displayPath = str_replace('\\', '/', $displayPath);


        echo sprintf("Path: %s, isDir: %d, isFile: %d\n",
            $displayPath,
            $item->isDir(),
            $item->isFile()
        );
    }
    echo "Count for $dirPath: $count\n\n";
}

?>
