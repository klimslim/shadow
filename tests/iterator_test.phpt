--TEST--
Check iterators failure
--SKIPIF--
<?php if (!extension_loaded("shadow")) print "skip"; ?>
--FILE--
<?php
require_once('setup.inc'); // This likely sets $template and $instance

// Corrected paths based on created fixtures
$topLevelDirs = [
    "$template/api", // Should be tests/fixtures/templatedir/api
    "$instance/qwe", // Should be tests/fixtures/instance/qwe
    "$instance/api", // Should be tests/fixtures/instance/api
];

foreach ($topLevelDirs as $dirPath) {
    echo "Iterating: $dirPath\n";
    if (!is_dir($dirPath)) {
        echo "Directory $dirPath does not exist!\n";
        continue;
    }
    $count = 0;
    $iterator = new RecursiveIteratorIterator(
        new RecursiveDirectoryIterator($dirPath, FilesystemIterator::SKIP_DOTS | FilesystemIterator::UNIX_PATHS),
        RecursiveIteratorIterator::SELF_FIRST
    );

    foreach ($iterator as $item) {
        $count++;
        // Use a relative path for EXPECTF robustness if $template/$instance changes base
        $displayPath = $item->getPathname();
        if (strpos($displayPath, $template) === 0) {
            $displayPath = '$template' . substr($displayPath, strlen($template));
        } elseif (strpos($displayPath, $instance) === 0) {
            $displayPath = '$instance' . substr($displayPath, strlen($instance));
        }
        // Normalize slashes
        $displayPath = str_replace('\\', '/', $displayPath);


        echo sprintf("Path: %s, isDir: %d, isFile: %d\n",
            $displayPath,
            $item->isDir(),
            $item->isFile()
        );
    }
    echo "Count for $dirPath: $count\n\n";
}

?>
--EXPECTF--
Iterating: %s/templatedir/api
Path: $template/api, isDir: 1, isFile: 0
Path: $template/api/rest.php, isDir: 0, isFile: 1
Count for %s/templatedir/api: 2

Iterating: %s/instance/qwe
Path: $instance/qwe, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks/.gitkeep, isDir: 0, isFile: 1
Count for %s/instance/qwe: 30

Iterating: %s/instance/api
Path: $instance/api, isDir: 1, isFile: 0
Path: $instance/api/rest.php, isDir: 0, isFile: 1
Count for %s/instance/api: 2
