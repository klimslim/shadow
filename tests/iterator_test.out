Iterating: /app/tests/fixtures/templatedir/api
Path: $template/api/rest.php, isDir: 0, isFile: 1
Count for /app/tests/fixtures/templatedir/api: 1

Iterating: /app/tests/fixtures/instance/qwe
Path: $instance/qwe/modulebuilder, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Contacts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Tasks/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Accounts/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/ExternalUsers/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Documents/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Project/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Cases/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Meetings/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Calls/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/PurchasedLineItems/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Notes/.gitkeep, isDir: 0, isFile: 1
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages, isDir: 1, isFile: 0
Path: $instance/qwe/modulebuilder/builds/qweqwe/SugarModules/modules/Messages/.gitkeep, isDir: 0, isFile: 1
Count for /app/tests/fixtures/instance/qwe: 29

Iterating: /app/tests/fixtures/instance/api
Path: $instance/api/rest.php, isDir: 0, isFile: 1
Count for /app/tests/fixtures/instance/api: 1